### PR TITLE
Live: use StreamingDataSource for live measurments

### PR DESCRIPTION
**Pull request by ryantxu**
extracted from #31010 to simplify that PR

### CODE REVIEW

**Review by torkelo — APPROVED** [review]
Looks good, 

Is the plan to migrate from arrow base64 to DataFrameJSON later?

**Comment by leeoniya** [comment]
yep

File: go.mod [hunk 1]
```diff
@@ -42,7 +42,7 @@ require (
 	github.com/grafana/alerting-api v0.0.0-20210311171115-b0eb4577f38c
 	github.com/grafana/grafana-aws-sdk v0.2.0
 	github.com/grafana/grafana-plugin-model v0.0.0-20190930120109-1fc953a61fb4
-	github.com/grafana/grafana-plugin-sdk-go v0.88.0
+	github.com/grafana/grafana-plugin-sdk-go v0.88.1-0.20210316010403-be8d0c26d32a
```
Comment (ryantxu) [inline go.mod:45]: This includes the FrameToJSON function... we should use a tagged release before merging!

File: packages/grafana-runtime/src/measurement/collector.ts [hunk 2]
```diff
@@ -1,212 +1,84 @@
...
   //------------------------------------------------------
   // Collector
   //------------------------------------------------------
 
-  addBatch = (batch: MeasurementBatch) => {
-    let action = batch.action ?? MeasurementAction.Append;
-    if (action === MeasurementAction.Clear) {
-      this.measurements.clear();
-      action = MeasurementAction.Append;
+  addBatch = (msg: MeasurementBatch) => {
```
Comment (ryantxu) [inline packages/grafana-runtime/src/measurement/collector.ts:65]: @FZambia this is a temporary hack while we sync the frontend and backend implementations.  Currently on the backend, we have types for `DataFrameJSON`, but not a batch of them -- we are also missing `key` on DataFrameJSON and that will be required if we want to support multiple frames in one channel

**Pull request merged**